setup-disk.in

```diff
@@ -994,6 +1006,21 @@ native_disk_install() {
 				"${root_size}${root_size:+M},$root_part_type" \
 				|| return 1
 
+		elif [ "$ARCH" = "s390x" ]; then
+			local blocks_per_track=12 tracks_per_cylinder=15
+			local boot_track=$((boot_size * 1024 / 4 / blocks_per_track))
+			local swap_track=$((swap_size * 1024 / 4 / blocks_per_track + boot_track + 1))
+			local conf="$(mktemp)"
+			# DASD devices do not support partition id
+			# assumptions were made that in 'sys' installation, partitions are in order: boot, swap, root
+			# TODO: mdev -s
+			cat > "$conf" <<- EOF
```

Comment:
Please don’t put space between `<<-` and `EOF`.

Reply:
Interesting. I have seen the differences between having space and not but can't remember it. Anyway it seems to work fine and I actually copied this from other place in the same file.